LightCore/Registration/RegistrationSource/EnumerableRegistrationSource.cs

```diff
@@ -31,15 +31,8 @@ internal class EnumerableRegistrationSource : IRegistrationSource
         {
             get
             {
-                return contractType => contractType.IsGenericEnumerable()
-                        &&
-                        ((this._registrationContainer.HasRegistration(
-                            contractType.GetGenericArguments().FirstOrDefault()))
-                         ||
-                         // Use ConcreteTypeRegistrationSource.
-                         (this._registrationContainer.IsSupportedByRegistrationSource(
-                             contractType.GetGenericArguments().FirstOrDefault())));
-
+                // Don't check the generic arguments, to ensure to create enumerables anyway.
+                return contractType => contractType.IsGenericEnumerable();

```

Comment:
So this is the real fix for ASP.NET Core Support. Because there are Enumerables demanded without any Enumerable Item Type registered within LightCore, right?

Reply:
Exactly :)

Reply:
The actual fix is done in the ReceiveInstanceInLifecycle, where I just needed to rearrange the code a little bit, to get an empty list returned in case of missing registrations.